.wa-env

```diff
@@ -169,9 +170,12 @@ export TEST_LAW_FIRM_SHARE_CASE_D_PASSWORD=London03
 export TEST_JUDGE_X_USERNAME=ia.iacjudge.ccd@fake.hmcts.net
 export TEST_JUDGE_X_PASSWORD=London20
 
-export WA_TASK_DMNS_BPMNS_PATH="your path to DMNs repo"
-export WA_BPMNS_DMNS_PATH="your path to wa-standalone-task-bpmn repo"
-export IA_TASK_DMNS_BPMNS_PATH="your path to ia-task-configuration repo"
-
+export WA_TASK_DMNS_BPMNS_PATH="$(dirname "$(mdfind kind:folder 'ia-docker' | head -1)")/wa-task-configuration-template"
+export WA_BPMNS_DMNS_PATH="$(dirname "$(mdfind kind:folder 'ia-docker' | head -1)")/wa-standalone-task-bpmn"
+export IA_TASK_DMNS_BPMNS_PATH="$(dirname "$(mdfind kind:folder 'ia-docker' | head -1)")/ia-task-configuration"
```

Comment:
The `mdfile` command is not normally available under Linux. Keep in mind that different suppliers use different machines and not everybody uses a Mac.

Replace it with a command that is available by default in Linux.

Reply:
replaced with a check if they're in profile and if not then to add them (and info what path needs to be added)